qiskit/scheduler/schedule_circuit.py

```diff
@@ -37,6 +37,8 @@ def schedule_circuit(circuit: QuantumCircuit,
           qubit resource. alias: ``'asap'``)
         * ``'as_late_as_possible'``: Schedule pulses late-- keep qubits in the ground state when
           possible. (alias: ``'alap'``)
+        * ``sequence``: Schedule trivially in an ASAP fashion, but use gate durations provided by
+                        the input schedule.
```

Comment:
does this need to be a "method"?

Reply:
No. I'll remove it from "method" of `schedule`.